modules/compliance-updating.adoc

```diff
@@ -52,3 +52,8 @@ workers-scan-no-empty-passwords   Applied
 ----
 
 . The nodes will apply the newer remediation version and reboot.
+
+[IMPORTANT]
```

Comment:
I wonder if there's something to be done here about the stacked admonitions? RHSSG advises to avoid this... its a very short module I understand. Maybe this particular one can move to the after the intro text for the module?